controllers/cloud.redhat.com/clowdenvironment_controller.go

```diff
@@ -75,30 +75,35 @@ func (r *ClowdenvironmentReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{Requeue: true}, fmt.Errorf("error setting annotations: %w", err)
 	}
 
-	ns, err := helpers.GetNamespace(ctx, r.Client, nsName)
+	namespace, err := helpers.GetNamespace(ctx, r.Client, nsName)
 	if err != nil {
-		r.Log.Error(err, "could not retrieve newly created namespace", "namespace", nsName)
+		r.Log.Error(err, "could not retrieve updated resource", "namespace", nsName)
```

Comment:
Should we be doing something else to handle this error? Bombing out or requeueing or something?